src/sidebar/app/app.js

```diff
@@ -36,9 +36,18 @@ browser.storage.local.get().then(result => {
   }
 
   const state = JSON.parse(result.redux || '{}');
+
   // We use stored state to propagate actions and avoid keeping
-  if (result.hasOwnProperty('credentials') && state.sync && state.sync.email) {
-    store.dispatch(authenticate(state.sync.email));
+  if (result.hasOwnProperty('credentials')) {
+    if (state.sync && state.sync.email) {
+      store.dispatch(authenticate(state.sync.email));
+    } else {
+      store.dispatch(authenticate('...'));
```

Comment:
`store.dispatch(authenticate('...')` the `...` looks strange, what does that do?

Reply:
UI is optimistic so we load footer as synced and tooltip display during few seconds 'Synced with ...'.

Reply:
ah cool